### PR TITLE
[WIP] Weekly Founder Recap Mail 

### PR DESCRIPTION
**Pull request by kuhnchris**
##### Description
See discussion over at #3013

### CODE REVIEW

**Comment by SaptakS** [comment]
Please run `make fix isort` to fix travis issues and push

**Comment by PixelantDesign** [comment]
Hey @kuhnchris please see comments from @SaptakS 

Aautomation sending: you will have to first create a management command for the same and then run the management command to the cron tasks.

Please fix linting. You'll need to run the command `make fix isort` to fix linting and for travis to pass.

**Comment by SaptakS** [comment]
We can always test it in staging without pushing into production to check how it looks. :) So I think it is better to get it in with the complete management command. Thanks.

**Comment by kuhnchris** [comment]
Sure, no problemo. Will attempt to implement it today then.

**Comment by kuhnchris** [comment]
One question remaining @PixelantDesign @SaptakS do we want to send "empty" bounty mails aswells, as in, "nothing changed, how about you put up a bounty" incentive, or do we just want to skip those people?

Thanks,
Chris

**Comment by SaptakS** [comment]
I think skip. But @PixelantDesign  would be a right person to take a call on it.

**Comment by PixelantDesign** [comment]
great suggestions @kuhnchris! I think we can open new tickets for those use cases.

File: app/retail/emails.py [hunk 1]
```diff
@@ -818,10 +906,13 @@ def render_new_bounty_roundup(to_email):
...
     return response_html, response_txt, subject
 
+# DJANGO REQUESTS
 
 
-
-# DJANGO REQUESTS
+@staff_member_required
+def weekly_recap(request):
+    response_html, _ = render_weekly_recap("mark.beacom@consensys.net")
```
Comment (thelostone-mc) [inline app/retail/emails.py]: @kuhnchris are we leaving this hardcoded here ?

**Comment by thelostone-mc** [comment]
@kuhnchris `weekly_founder_recap.txt` -> It's an empty file
Could we populate that with just data ?

Also hardcoding @mbeacom. Shouldn't we remove that? 

Could you address the stickler issues please ?

File: app/retail/emails.py (hunk 1 above)
Comment (kuhnchris) [reply app/retail/emails.py]: This is just for /_administration/, we can put in someone else if you want, we just need someone who had regulary bounty activities so it outputs "something".

**Comment by kuhnchris** [comment]
Ah the .txt one is for the non-html one, right? sure I can add a bit of content there.
Regarding mbeacom, it's just for administration/admin, we can add someone else if you have any suggestion.
addressing stickler, sure.

**Comment by thelostone-mc** [comment]
@kuhnchris Ah it's good ^_^
will merge after CI goes green

**Comment by kuhnchris** [comment]
ci won't go green. ever. until we fix it.

**Pull request merged**